### ISSUE TITLE
Pinch to zoom on touch screen devices

### ISSUE THREAD

**Issue by verhovsky** [floooh/v6502r#18]
doesn't work.

**Comment by floooh** [comment]
Yeah,doesn't work indeed. There's no high level gesture recognition in sokol_app.h, only the low level begin/move/end/cancel events. Gesture recognition would need to be built on top of those low level events.

**Comment by floooh** [comment]
Deleted my previous comment since I didn't notice that this is in the v6502r repo, and not in the sokol repo.
